docs/sql-reference/sql-functions/date-time-functions/makedate.md

```diff
@@ -0,0 +1,72 @@
+# makedate
+
+## Description
+
+Reduces the specified date and time by a specified number of hours.
+
+## Syntax
+
+```Haskell
+DATE makedate(INT year, INT dayOfYear);
+```
+
+## Parameters
+
+* `year`: Year ranges from 0 to 9999.The supported data type is INT. 
+
+* `dayOfYear`: The day of the year. The supported data type is INT.
+
+## Return value
+
+Returns a date, given year and day-of-year values.
```

Comment:
Returns the date on the dayOfYear-th day of the given year，I think this description is more accurate